lib/debug.ml

```diff
@@ -29,7 +29,7 @@ let get_thread_id () =
 
 module IntMap = Map.Make(struct
   type t = int
-  let compare = Pervasives.compare
+  let compare = Stdlib.compare
```

Comment:
Not that it matters here, but you can get smaller code and better performance by explicitly telling the compiler that your arguments are ints. Otherwise it will use the generic value comparator. See https://ocaml.godbolt.org/z/JS32TL for the difference.

Reply:
Do you mean using `Map.Make(Int)`?

Reply:
Ah yes now that we have an Int module we could just do it that way.